File: string/2027_minimum_moves_to_convert_string/2027_minimum_moves_to_convert_string.go
package main

func minimumMoves(s string) int {
	result := 0
	for i := 0; i < len(s); {
		if s[i] == 'X' {
			result += 1
			i += 3
		} else {
			i += 1
		}
	}
	return result
}

File: string/2027_minimum_moves_to_convert_string/2027_minimum_moves_to_convert_string.py
class Solution:
    def minimumMoves(self, s: str) -> int:
        l = len(s)
        i = 0
        result = 0
        while i < l:
            if s[i] == 'X':
                result += 1
                i += 3
            else:
                i += 1
        return result
